### PR TITLE
BM16a stock layout

### PR DESCRIPTION
**Pull request by jdarowski**
## Description

Upon receiving the BM16a i decided to make a keymap of my own (obviously), but wanted to have a fallback in case I don't like it and I might not be the only one, so here it is. The stock keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Nothing in particular

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bm16a/keymaps/stock/readme.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# The default keymap for bm16a
```
Comment (drashna) [inline keyboards/bm16a/keymaps/stock/readme.md]: Might want to change the readme or remove it

File: keyboards/bm16a/keymaps/stock/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019
```
Comment (fauxpark) [inline keyboards/bm16a/keymaps/stock/config.h]: This file can be deleted.

File: keyboards/bm16a/keymaps/stock/keymap.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    KC_P7,    KC_P8,    KC_P9,    KC_PMNS,  \
+    KC_P4,    KC_P5,    KC_P6,    KC_PPLS,  \
+    KC_P1,    KC_P2,    KC_P3,    KC_PENT,  \
+    KC_P0,    KC_PDOT,  KC_SPC,   MO(_FN1)  \
+  ),
+  [_FN1] = LAYOUT_ortho_4x4(
+    RESET,    KC_PAST,  KC_PSLS,  _______,  \
+    BL_TOGG,  BL_DEC,   BL_INC,   BL_STEP,  \
+    RGB_TOG,  RGB_MOD,  RGB_HUI,  RGB_HUD,  \
+    RGB_SAI,  RGB_SAD,  _______,  _______   \
```
Comment (fauxpark) [inline keyboards/bm16a/keymaps/stock/keymap.c]:
```suggestion
    KC_P7,    KC_P8,    KC_P9,    KC_PMNS,
    KC_P4,    KC_P5,    KC_P6,    KC_PPLS,
    KC_P1,    KC_P2,    KC_P3,    KC_PENT,
    KC_P0,    KC_PDOT,  KC_SPC,   MO(_FN1)
  ),
  [_FN1] = LAYOUT_ortho_4x4(
    RESET,    KC_PAST,  KC_PSLS,  _______,
    BL_TOGG,  BL_DEC,   BL_INC,   BL_STEP,
    RGB_TOG,  RGB_MOD,  RGB_HUI,  RGB_HUD,
    RGB_SAI,  RGB_SAD,  _______,  _______
```

File: keyboards/bm16a/keymaps/stock/config.h (hunk 2 above)
Comment (jdarowski) [reply keyboards/bm16a/keymaps/stock/config.h]: Will do.

File: keyboards/bm16a/keymaps/stock/readme.md (hunk 1 above)
Comment (jdarowski) [reply keyboards/bm16a/keymaps/stock/readme.md]: I was merely following the readme in the default keymap. If you dislike it then I'll gladly delete it, but then users might fail to understand the reason for this keymap's existence.

Comment (noroadsleft) [reply keyboards/bm16a/keymaps/stock/readme.md]: Maybe a

```diff
diff --git a/keyboards/bm16a/keymaps/factory/readme.md b/keyboards/bm16a/keymaps/factory/readme.md
index b5dc5e731..148cf29af 100644
--- a/keyboards/bm16a/keymaps/factory/readme.md
+++ b/keyboards/bm16a/keymaps/factory/readme.md
@@ -1,4 +1,4 @@
-# The default keymap for bm16a
+# The factory keymap for bm16a

-This is the default layout this keypad ships with. Use it if you want to return to stock.
+This is the layout this keypad ships with. Use it if you want to return to stock.
```

and a 

```
git mv -- keyboards/bm16a/keymaps/stock/ keyboards/bm16a/keymaps/factory/
```

?

Comment (zvecr) [reply keyboards/bm16a/keymaps/stock/readme.md]: Its more "stock" like, as we have no audit trail of what the board actually shipped with. Returning to stock would likely involve dumping some fimware from a non QMK flashed board, or begging kprepublic for the original fimware.

Comment (jdarowski) [reply keyboards/bm16a/keymaps/stock/readme.md]: Thank you @noroadsleft. I initially named the keymap "factory", but later changed it to stock. I reverted it to be what it was.

Those are some valid points @zvecr. I changed the readme accordingly.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**